Makefile

```diff
@@ -62,3 +62,14 @@ endif
 .PHONY: manpages
 manpages:
 	go run ./cmd/gen-docs --man-page --doc-path ./share/man/man1/
+
+DESTDIR :=
+prefix  := /usr/local
+bindir  := ${prefix}/bin
+.PHONY: install install-strip install-dirs
+install-dirs:
+	install -d ${DESTDIR}${bindir}
+install-strip: install-dirs bin/gh
```

Comment:
Could you elaborate on why separate targets `install-dirs` and `install-strip` are useful?

Reply:
Directories need to be created by all install targets, that's the only reason to have a `install-dirs` target.

`install-strip` is provided by autotools, so I thought it may be a good idea to support that too.